src/main/js/account/account.js

```diff
@@ -1,8 +1,61 @@
 import React from "react";
 import Container from "react-bootstrap/Container";
+import {Table} from "react-bootstrap";
 
+/**
+ * The purpose of this method is to generate teh bank account table
+ *
+ * @author Xiaobing Hou
+ * @date 06/03/2022
+ */
 const Account = (props) => {
-  return <Container>Accounts</Container>;
+
+    let total = 0;
+    for (let i = 0; i < props.bankAccount.length; i++) {
+        total = total + props.bankAccount[i].balance;
+    }
+    total = total.toFixed(2);
```

Comment:
Instead of having inline code like this create functions.

Reply:
Done